app/views/orders/_form.html.haml

```diff
@@ -0,0 +1,52 @@
+= simple_form_for order, url: { action: :update_or_purchase } do |f|
+  %table.table.table-striped.table-hover#cart
+    %thead
+      %tr
+        %th
+        %th= t(".th.product")
+        %th.centered= t(".th.quantity")
+        %th.currency= t(".th.cost")
+        - if order.has_subsidies?
+          %th.currency= t(".th.adjust")
+        %th.currency= t(".th.extend")
+
+    %tbody
+      - prev_group_id = nil
+      - order.order_details.order(:group_id).each do |order_detail|
+        = render "cart_row",
+          order_detail: order_detail,
+          first_of_group: order_detail.group_id && (prev_group_id != order_detail.group_id)
+        - prev_group_id = order_detail.group_id
+
+      %tr
+        %td.currency{colspan: "3"}
+          %b= t(".td.total")
+        %td.currency
+          %b= number_to_currency order.estimated_cost
+        - if order.has_subsidies?
+          %td.currency
+            %b= number_to_currency order.estimated_subsidy
+        %td.currency
+          %b= number_to_currency order.estimated_total
+
+  %p.footnote= t(".foot.all")
+
+  - if order.instrument_order_details.count > 0
+    %p.footnote= t(".foot.instrument")
+
+  - if order.order_details.size > 0
+    %ul.inline{style: "float: left"}
+      %li= submit_tag t("shared.update"), class: "btn"
+      - if order.validated?
+        %li
+          = f.submit t("shared.purchase"),
+            class: %w(btn btn-primary),
+            confirm: t(".confirm")
+
+      %li
+        = render_view_hook "acting_as",
```

Comment:
This might be related to the cleaning up the styling, but this is displaying even if you aren't "ordering as"